opentracing/utils.go

```diff
@@ -0,0 +1,27 @@
+package opentracing
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+)
+
+const (
+	OpenTracingContextIDHeader = "OpenTracing-Context-Id"
```

Comment:
Q: is there any value in the standard header name? We're having an internal debate about it. IMO, if two services are running with two different DCP implementations, the server service is not going to be able to make sense of the header sent by the client service, therefore having a universal header name is counterproductive. So maybe the header name should be an implementation detail of the Recorder?

One case where the common header might be useful is if the two tracing systems have the exact same representation and meaning of the context ID, but that seems rather contrived.